src/Analyzers/CSharp/Tests/FileHeaders/FileHeaderTests.cs

```diff
@@ -378,10 +388,6 @@ namespace Bar
             var fixedCode = @"// Copyright (c) SomeCorp. All rights reserved.
 // Licensed under the ??? license. See LICENSE file in the project root for full license information.
 
-/* Copyright (c) OtherCorp. All rights reserved.
- * Licensed under the ??? license. See LICENSE file in the project root for full license information.
- */
-
```

Comment:
shoudl the test name be updated at all here?

Reply:
The existing wrong block comment gets replaced by the new text template in single line comment style. The only difference is, that before the fixer added the new header, while it now replaces the header. The test name does not say anything about the replacement behavior, so it is still fine.

The `<summary>` of most of the existing tests are misleading, though and should be removed.